app/stores/NodeUpdateStore.js

```diff
@@ -40,6 +41,7 @@ export default class NodeUpdateStore extends Store {
   };
 
   @action _postponeNodeUpdate = () => {
+    this.postponeUpdateRequest.execute();
```

Comment:
We should probably add some basic error handling here! Like
```javascript
@action _postponeNodeUpdate = async () => {
  try {
    await this.postponeUpdateRequest.execute();
    this.isUpdatePostponed = true;
  } catch(error) {
    this.isUpdatePostponed = false;
  }

Reply:
Agreed @DominikGuzei - the only part that worries me is the fact that we should add "submitting" state on postpone button AND handle error state in the UI as if this call fails we will not hide the update notification so the user should be noticed that the error happened...

Reply:
Yeah definitely, the whole error handling logic needs to be fleshed out for the UI